python/res/enkf/ecl_config.py

```diff
@@ -26,34 +26,35 @@
 class EclConfig(BaseCClass):
     TYPE_NAME = "ecl_config"
 
-    _alloc                  = ResPrototype("void* ecl_config_alloc(config_content)", bind = False)
```

Comment:
What happened here?

Reply:
Formatting so that it looks a bit better. 

 _get_last_history_restart = ResPrototype("int ecl_config_get_last_history_restart(ecl_config)") 

Didnt fit on the same column